hq_superset/services.py

```diff
@@ -143,6 +152,48 @@ def to_sql(df, replace=False):
         raise ex
 
 
+def subscribe_to_hq_datasource(domain, datasource_id):
+    client = _get_or_create_oauth2client(domain)
+    hq_request = HQRequest(url=datasource_subscribe(domain, datasource_id))
+    webhook_url = current_app.url_for(
+        'DataSetChangeAPI.post_dataset_change',
+        _external=True,
+    )
+    token_url = current_app.url_for('OAuth.issue_access_token', _external=True)
```

Comment:
(related to [this](https://github.com/dimagi/commcare-hq/pull/34179#issuecomment-1985185551) comment)

[This](https://stackoverflow.com/questions/66663507/flask-url-for-http-instead-of-https) is an interesting post. It seems like one might need to explicitly specify the `https` scheme should be used.

Reply:
9042fb6e37d1e5a2f9d9a19e264f62c615ce09fd